### PR TITLE
feat: Rest API Documentation and Initial version of Post and Get Invitations

### PR DESCRIPTION
**Pull request by rolsonquadras**
part of #8 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: 2020/01/aries-framework-go/cmd/lobby/main.go [hunk 1]
```diff
@@ -13,14 +13,21 @@ import (
...
 	"github.com/trustbloc/presentation/2020/01/aries-framework-go/cmd/lobby/store/mem"
 )
 
-const addressPattern    = ":%s"
+const (
+	addressPattern    = ":%s"
+	demoPath   = "/demo"
+	getDemoPath   = demoPath + "/{uid}"
+	postInvitationsPath   = "/invitation/{uid}"
+	getInvitationsPath   = "/invitations"
```
Comment (troyronda) [inline 2020/01/aries-framework-go/cmd/lobby/main.go]: you should be getting the invitation for a specific demo ID.

Comment (rolsonquadras) [reply 2020/01/aries-framework-go/cmd/lobby/main.go]: [Updated](https://github.com/trustbloc/presentation/compare/428aeb5046c1724122d6624f8fdf55b0a3ee172c..62022998693fad6138c4ff039c75831db8f237b0#diff-281725cdb6328d7bb6e6dd0a31d6f280R178)

File: 2020/01/aries-framework-go/cmd/lobby/api/model.go [hunk 2]
```diff
@@ -10,3 +10,18 @@ package api
...
 	ID string `json:"id,omitempty"`
 }
+
+type Invitations struct {
+	DIDExchangeInvitations []Invitation `json:"invitations,omitempty"`
+}
+
+
+type Invitation struct {
+	Type   			string 		`json:"@type,omitempty"`
```
Comment (troyronda) [inline 2020/01/aries-framework-go/cmd/lobby/api/model.go]: what happened to the formatting?

Comment (rolsonquadras) [reply 2020/01/aries-framework-go/cmd/lobby/api/model.go]: I'll fix it in next push.

**Pull request merged**